app/assets/javascripts/router.js

```diff
@@ -243,7 +243,7 @@ class ReactRouter extends React.Component<Props> {
                 isAuthenticated={isAuthenticated}
                 path="/datasets/upload"
                 render={() => (
-                  <DatasetUploadView
+                  <DatasetAddView
```

Comment:
You can remove the `onUploaded` prop here as you copied it to the dataset_add_view.